src/App/ActivityList.purs

```diff
@@ -214,21 +163,38 @@ priorityToColor priority =
     Medium -> "#f5f588"
     Low -> "#469dd0"
 
-type Slots = ( addTodoDialog :: forall query . H.Slot query AddTodoDialog.Output Int )
+type Slots = ( addTodoDialog :: forall query. H.Slot query AddTodoDialog.Output Int
+             , tagModal :: forall query. H.Slot query TagModal.Output Int
+             )
 
 _addTodoDialog = Proxy :: Proxy "addTodoDialog"
+_tagModal = Proxy :: Proxy "tagModal"
 
 render :: forall m. State -> H.ComponentHTML Action Slots m
 render state =
   HH.div [ Prop.class_ (ClassName "columns")]
     [ sidebarView state
     , panelsView state
-    , modalView state
     , if state.showAddTodoModal
       then HH.div_ [ HH.slot _addTodoDialog 0 addTodoDialog 0 HandleAddTodo ]
       else HH.div_ []
+    , case state.modalTarget of
+        Just task -> HH.div_ [ HH.slot _tagModal 1 TagModal.tagModal task HandleTags ]
+        Nothing -> HH.div_ []
     ]
 
+-- slot
+--   :: forall query action input output slots m label slot _1
+--    . Cons label (Slot query output slot) _1 slots
+--   => IsSymbol label
+--   => Ord slot
+--   => Proxy label
+--   -> slot
+--   -> Component query input output m
+--   -> input
+--   -> (output -> action)
+--   -> ComponentHTML action slots m
+
```

Comment:
@tmciver  Assuming this a carryover to have a ready type definition fr the slot. Since the type is so complex, its a good inline resource, especially if the LSP is not working correctly..